CONTRIBUTING.md

```diff
@@ -65,21 +65,54 @@ Most of the required dependencies can be installed via *pip*, using the
 
 ### Linux
 
-Qiskit is supported on RedHat (CentOS, Fedora) and Ubuntu based distributions, as long as you can install a GCC version that is C++14 compatible.
+Qiskit is officially supported on Red Hat, CentOS, Fedora and Ubuntu distributions, as long as you can install a GCC version that is C++14 compatible and the few dependencies we need.
 
-Ubuntu prerequisites
+To get most of the necessary compilers and libraries, install the *development environment* tools from your Linux distribution by running
 
-To get most of the necessary compilers
-and libraries, install the *build-essential* package by running
+CentOS/Red Hat
+
+    $ yum groupinstall "Development Tools"
```

Comment:
From centos8 you can just substitute `yum` by `dnf`. As `yum` can still be used I don't know if adding this info is worth it, though.